test/testobjects/TestOrchestrations.ts

```diff
@@ -306,12 +306,12 @@ export class TestOrchestrations {
     });
 
     public static ThrowsExceptionFromActivity: any = df.orchestrator(function*(context: any)
-    : IterableIterator<unknown> {
+    : Generator<void, void, any> {
```

Comment:
I realize you are just editing the type signature to be correct here, but is there a reason we need the type signature here? Many of the other orchestrations in this file are missing them.

Reply:
Good observation. I automatically changed the `IterableIterator` to `Generator` because, in the past, this test had been given a type for some reason. 

But removing the type completely still type-checks, so I'm removing it. Thanks for pointing it out!